app/controllers/tool_groups_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ToolGroupsController < ApplicationController
+  before_action :authorize!
+
+  def index
+    render json: ToolGroup.all.order(name: :asc), status: :ok
+  end
+
+  def create
```

Comment:
when making this call:
```
curl --location 'https://mobile-content-api-stage.cru.org/tool-groups' \
--header 'Accept: application/vnd.api+json' \
--header 'Content-Type: application/vnd.api+json' \
--data '{
 "data": {
  "type": "tool-group",
  "attributes": {
   "name": "Tool Group Name 3",
   "suggestions-weight": "1.0"
  }
 }
}'
```
I get an error "suggestions_weight can't be blank".

is it possible to have it be consistent with how we spell `suggestions-weight` elsewhere using the `-`? I'm not sure if we've ever actually made this consistent elsewhere within the API before. maybe just do some standardization of attribute names (`tr('-', '_')`) when they are being parsed by the API to create/update a data model?

Reply:
we can discuss this if it'll be too convoluted to standardize it here.

The higher importance for me in consistency for attribute names/relationship names & paths across the API

Reply:
maybe this logic can be extracted to a common `ToolGroupsController#permitted_params` method since it will be used for both creates and updates.

e.g. https://github.com/CruGlobal/mobile-content-api/blob/master/app/controllers/resources_controller.rb#L72-L74

Reply:
Daniel, I've implemented a [method](https://github.com/CruGlobal/mobile-content-api/commit/435a5ba6a8e60fc53daeffd6652d2943a6fdd0bd#diff-766c34fd6533171eaf54300c153f89d6002c35c02cfc9c5b219251f85180ad07R123) that is available for all controllers and converts the hyphens into underscore.

Now we can do a POST request like this

```
curl --location 'http://localhost:3000/tool-groups' \
--header 'Content-Type: application/vnd.api+json' \
--header 'Accept: application/vnd.api+json' \
--data '{
 "data": {
  "type": "tool-group",
  "attributes": {
   "name": "Test Group Name 10",
   "suggestions-weight": "1.0"
  }
 }
}'
```

or  a PUT request like this

```
curl -X PUT --location 'http://localhost:3000/tool-groups/10' \
--header 'Content-Type: application/vnd.api+json' \
--header 'Accept: application/vnd.api+json' \
--data '{
 "data": {
  "type": "tool-group",
  "attributes": {
   "name": "Test Group Name 10.1",
   "suggestions-weight": "1.1"
  }
 }
}'
```

Reply:
cool, I'll get back to testing this shortly, in the middle of a couple other tasks